charts/opentelemetry-operator/values.schema.json

```diff
@@ -612,7 +612,9 @@
                     "required": [
                         "ENABLE_WEBHOOKS"
                     ],
-                    "additionalProperties": false,
+                    "additionalProperties": {
+                      "type": "string"
+                    },
```

Comment:
Lets not even restrict it like this.  I think the entiure section should be:

```json

                "env": {
                    "type": "object",
                    "examples": [{
                        "ENABLE_WEBHOOKS": "true"
                    }]
                },
```

Reply:
I've changed it as you shared, but let me know your opinion on the case I mentioned above :-)

Reply:
I can already make the deployment.yaml env dynamic
```
          {{- if .Values.manager.env }}
          env:
            {{- range $name, $value := .Values.manager.env }}
            - name: {{ $name }}
              {{- if eq (kindOf $value) "map" }}
              {{- tpl ($value | toYaml) $ | nindent 14 }}
              {{- else }}
              value: {{ tpl $value $ | quote -}}
              {{- end }}
            {{- end }}
          {{- end }}
```

```
manager:
  env:
    WATCH_NAMESPACE:
      valueFrom:
        fieldRef:
          fieldPath: metadata.namespace

```

helm template -s templates/deployment.yaml -f values-test.yaml ./ | grep -A 10 env
```
          env:
            - name: ENABLE_WEBHOOKS
              value: "true"
            - name: WATCH_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
          image: "ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:0.96.0"
          name: manager
```

Reply:
You're right, this field isn't following the same pattern as the collector chart. The schema should restrict it to any number of `key: value` pairs where `key` and `value` are strings

Reply:
done. that should do it